### PR TITLE
Integrate appraisal tab

### PR DESCRIPTION
**Pull request by jraddaoui**

### CODE REVIEW

File: src/dashboard/src/components/filesystem_ajax/views.py [hunk 1]
```diff
@@ -658,46 +661,69 @@ def copy_files_to_arrange(sourcepath, destination):
             logger.exception('Integrity error inserting: %s', entry)
 
 
-def copy_to_arrange(request, sourcepath=None, destination=None):
-    """ Add files to in-progress SIPs being arranged.
+def copy_to_arrange(request, sources=None, destinations=None, fetch_children=False):
```
Comment (jrwdunham) [inline src/dashboard/src/components/filesystem_ajax/views.py:676]: Docstring should be modified to match the new function signature.

File: src/dashboard/src/components/filesystem_ajax/views.py [hunk 2]
```diff
@@ -658,46 +661,69 @@ def copy_files_to_arrange(sourcepath, destination):
...
+    # Possibly the constant used to refer to externally-constructed
+    # paths and the one used solely internally should be two different
+    # constants.
+    if sources[0].startswith('/' + DEFAULT_BACKLOG_PATH):
+        action = 'copy'
+        backlog_uuid = storage_service.get_location(purpose='BL')[0]['uuid']
+    elif sources[0].startswith(DEFAULT_ARRANGE_PATH):
+        action = 'move'
+    else:
+        logger.error('Filepath %s is not in base backlog path nor arrange path')
```
Comment (jrwdunham) [inline src/dashboard/src/components/filesystem_ajax/views.py]: Nothing corresponding to '%s' here ...

File: src/dashboard/src/components/filesystem_ajax/views.py [hunk 3]
```diff
@@ -658,46 +661,69 @@ def copy_files_to_arrange(sourcepath, destination):
...
+                    fetch_children=fetch_children, backlog_uuid=backlog_uuid)
+                response = {'message': 'Files added to the SIP.'}
+                status_code = 201
+            elif action == 'move':
+                move_files_within_arrange(source, dest)
+                response = {'message': 'SIP files successfully moved.'}
+                status_code = 200
     except ValueError as e:
-        logger.exception('copy_to_arrange failed copying %s to %s', (sourcepath, destination))
+        logger.exception('Failed copying %s to %s', (source, dest))
```
Comment (jrwdunham) [inline src/dashboard/src/components/filesystem_ajax/views.py]: I think the syntax is off here, no? Shouldn't it be one of the following:

```
`logger.exception('Failed copying %s to %s', source, dest)`
`logger.exception('Failed copying %s to %s' % (source, dest))`
```

**Review by jrwdunham — COMMENTED** [review]
I found two potential issues with `%s` in log calls and a small docstring issue. Otherwise looks good to me.

**Comment by Hwesta** [comment]
Merged #502 into this branch before it's merged to qa/1.x

File: src/dashboard/src/components/filesystem_ajax/views.py (hunk 2 above)
Comment (Hwesta) [reply src/dashboard/src/components/filesystem_ajax/views.py]: Fixed.

File: src/dashboard/src/components/filesystem_ajax/views.py (hunk 3 above)
Comment (Hwesta) [reply src/dashboard/src/components/filesystem_ajax/views.py]: Good catch, fixed.

File: src/dashboard/src/components/filesystem_ajax/views.py (hunk 1 above)
Comment (Hwesta) [reply src/dashboard/src/components/filesystem_ajax/views.py:676]: Renamed parameters in docstring.

**Pull request merged**